DIRECTORY.md

```diff
@@ -499,6 +499,7 @@
     * [Soln](https://github.com/TheAlgorithms/Python/blob/master/project_euler/problem_30/soln.py)
   * Problem 31
     * [Sol1](https://github.com/TheAlgorithms/Python/blob/master/project_euler/problem_31/sol1.py)
+    * [Sol2](https://github.com/TheAlgorithms/Python/blob/master/project_euler/problem_31/sol2.py)
```

Comment:
As discussed in CONTRIBUTING.md, please do not modify this file as it is auto-generated.